examples/customs items/retrieve_customs_item

```diff
@@ -0,0 +1,7 @@
+require 'easypost'
```

Comment:
Issue: This file has a space in the name which is invalid and does not end with `.rb` which it requires.